service/models.py

```diff
@@ -135,12 +135,13 @@ def find(cls, by):
         logger.info("Processing lookup for id %s and condition %s ...", by_id, by_condition)
         return cls.query.get((by_id, by_condition))
 
-    @classmethod
-    def find_by_name(cls, name):
-        """Returns all Inventories with the given name
-
-        Args:
-            name (string): the name of the Inventories you want to match
-        """
-        logger.info("Processing name query for %s ...", name)
-        return cls.query.filter(cls.name == name)
+    #uncomment for sprint 2
```

Comment:
In commented code is unused, can consider removing it.

Reply:
Yeah, was thinking about it, but we will require it in the next sprint, and this ensure a code coverage of 91% for models.py

Reply:
Alright!